ironfish/src/wallet/scanner/walletScanner.ts

```diff
@@ -105,9 +105,12 @@ export class WalletScanner {
       this.state = scan
       unlock()
 
```

Comment:
nit:
```
logScanState && this.logger.debug(
  `Scan starting from block ${scan.start.sequence} to ${scan.end.sequence}`,
)
```

Reply:
eslint places `logScanState &&` on a separate line:
```
logScanState &&
        this.logger.debug(
          `Scan starting from block ${scan.start.sequence} to ${scan.end.sequence}`,
        )
```

I think the `if` statement is a little clearer, and it's only adding one line for the closing bracket `}`